lib/stencil-push.utils.js

```diff
@@ -89,15 +89,23 @@ utils.getThemes = (options, callback) => {
 
 utils.generateBundle = (options, callback) => {
     let bundle;
+    let output = {
+        dest: os.tmpdir(),
+        name: uuid(),
+    };
 
     if (options.bundleZipPath) {
         return async.nextTick(callback.bind(null, null, options));
     }
 
-    bundle = new Bundle(themePath, themeConfig, themeConfig.getRawConfig(), {
-        dest: os.tmpdir(),
-        name: uuid(),
-    });
+    if (options.saveBundleName) {
+        output = {
+            dest: themePath,
+            name: options.saveBundleName
```

Comment:
You are missing a trailing comma here which is causing the build to fail

Reply:
Fixed, seems good now